python/cudf/cudf/tests/test_orc.py

```diff
@@ -724,6 +724,105 @@ def test_orc_write_statistics(tmpdir, datadir, nrows, stats_freq):
                 assert stats_num_vals == actual_num_vals
 
 
+@pytest.mark.parametrize("stats_freq", ["STRIPE", "ROWGROUP"])
+@pytest.mark.parametrize("nrows", [2, 100, 6000000])
+def test_orc_chunked_write_statistics(tmpdir, datadir, nrows, stats_freq):
```

Comment:
I ran into this during a refactor, `datadir` and `stats_freq` aren't being used in this pytest. Do we want to keep them or remove them @hyperbolic2346 ?